src/xwayland/xserver.rs

```diff
@@ -159,10 +159,10 @@ fn launch<WM: XWindowManager + 'static>(inner: &Rc<RefCell<Inner<WM>>>) -> Resul
         guard
             .wayland_display
             .borrow_mut()
-            .create_client(wl_me.into_raw_fd())
+            .create_client(wl_me.into_raw_fd(), &mut ())
```

Comment:
No idea what the extra argument is for, but it wanted a`&mut impl Any` and I gave it one.

Reply:
This is what will be used as `DispatchData`, as such it should be provided by the user and forwarded here.

Reply:
Is this at all related to `XWayland::init`s `Data` type parameter?